### PR TITLE
add check for core commands on create formula

### PR DESCRIPTION
**Pull request by victor-schumacher**
**- What I did**
Added a check for core commands on create formula

**- How to verify it**
Create a formula with core command verb on it

**- Description for the changelog**
Added a check for core commands on create formula

### CODE REVIEW

File: pkg/cmd/create_formula.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ var (
 )
 
 const notAllowedChars = `\/><,@`
+var notAllowedWords = []string{"tutorial"}
```
Comment (kaduartur) [inline pkg/cmd/create_formula.go:44]: I think that you can use the [**CoreCmds**](https://github.com/ZupIT/ritchie-cli/blob/master/pkg/api/api.go) variable to **notAllowedWords**

Comment (victor-schumacher) [reply pkg/cmd/create_formula.go]: yes, i'm using it now. Thanks!

**Review by lucasdittrichzup — APPROVED** [review]

File: pkg/cmd/create_formula.go [hunk 2]
```diff
@@ -152,6 +157,22 @@ func (c createFormulaCmd) runPrompt() CommandRunnerFunc {
...
+				"Example: rit aws list bucket\n",
+				api.CoreCmds[i].Usage)
+
+			return errors.New(errorString)
+		}
+	}
+	return nil
+}
+
+
```
Comment (brunasilvazup) [inline pkg/cmd/create_formula.go:175]: Extra line?

Comment (victor-schumacher) [reply pkg/cmd/create_formula.go]: A intruder.
Removed.
Thanks.

File: pkg/cmd/create_formula.go [hunk 3]
```diff
@@ -105,6 +106,10 @@ func (c createFormulaCmd) runPrompt() CommandRunnerFunc {
 			return ErrNotAllowedCharacter
 		}
 
+		if err := coreCmdVerify(formulaCmd); err != nil {
+			return err
+		}
+
```
Comment (brunasilvazup) [inline pkg/cmd/create_formula.go:112]: I think we lacked to add the validation function in the `runStdin`  too.

Comment (victor-schumacher) [reply pkg/cmd/create_formula.go:112]: That's true, I'm adding it

**Review by lucasdittrichzup — APPROVED** [review]

**Pull request merged**